DESCRIPTION

```diff
@@ -67,7 +67,11 @@ Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
 Suggests: 
     covr,
-    testthat (>= 3.0.0)
+    testthat (>= 3.0.0),
+    remotes,
```

Comment:
{remotes} will eventually be taken over by {pak} although the data has yet to be announced.